.github/workflows/maven-postgres-tests-build-skip.yml

```diff
@@ -31,7 +31,7 @@ permissions:
   checks: write
 
 concurrency: 
-  group: maven-build-${{ github.head_ref || github.run_id }}
+  group: maven-postgres-tests-skip-build-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
```

Comment:
Let's remove the concurrency from the skip tests!

Reply:
Removed concurrency from maven-postgres-tests-build-skip.yml